.github/workflows/windows-app-test.yml

```
@@ -0,0 +1,56 @@
+name: Windows CI
+on: [pull_request]
+
+jobs:
+  run-windows-tests:
+    name: Build & run tests
+    runs-on: windows-2019
+
+    steps:
+    - uses: actions/checkout@v2
+      name: Checkout Code
+
+    - name: Setup Node.js
+      uses: actions/setup-node@v1
+      with:
+        node-version: '12.9.1'
+
+    - name: Setup MSBuild
+      uses: microsoft/setup-msbuild@v1.0.0
```

out of curiosity, why pin the patch version even?
```suggestion
      uses: microsoft/setup-msbuild@v1
```

should be sufficient and allow bugfixes?

---

Thank you. No issues here.
I've added the suggested change to the latest commit.